frontend/www/js/omegaup/components/qualitynomination/List.test.ts

```diff
@@ -0,0 +1,62 @@
+import { shallowMount } from '@vue/test-utils';
+import expect from 'expect';
+import Vue from 'vue';
+
+import T from '../../lang';
+import * as UI from '../../ui';
```

Comment:
```suggestion
import * as ui from '../../ui';
```

para que sea consistente con los otros lugares donde importamos esto.

Reply:
listo

Reply:
listo